internal/handshake/session_ticket.go

```diff
@@ -21,10 +21,13 @@ func (t *sessionTicket) Marshal() []byte {
 	b := make([]byte, 0, 256)
 	b = quicvarint.Append(b, sessionTicketRevision)
 	b = quicvarint.Append(b, uint64(t.RTT.Microseconds()))
+	if t.Parameters == nil {
```

Comment:
Should we bump the `sessionTicketRevision`? Not sure if it's actually needed, but we'd be on the safe side.

Reply:
It has been bumped to 4.